textile.go

```diff
@@ -324,6 +327,7 @@ func (x *initCmd) Execute(args []string) error {
 		CafeOpen:        x.CafeOptions.Open,
 		CafeURL:         envOrFlag("CAFE_HOST_URL", x.CafeOptions.URL),
 		CafeNeighborURL: envOrFlag("CAFE_HOST_NEIGHBOR_URL", x.CafeOptions.NeighborURL),
+        // ^ @todo why do we prefer env over flag for these? shouldn't flag override the env?
```

Comment:
Good point. Flag should take precedence.

Reply:
Ok, will merge then open a new PR.

Reply:
https://github.com/textileio/go-textile/pull/732